static/styles/landing-page.css

```diff
@@ -343,24 +343,14 @@ nav ul li.active::after {
 
     padding-top: 120px;
 
-    transition: all 0.5s ease;
-    -webkit-transform: translateY(120px);
-    transform: translateY(120px);
+    transition: opacity 0.5s ease;
```

Comment:
I disagree that it makes the experience feel slow. I think it looks more smooth.

![transition](https://user-images.githubusercontent.com/10321399/29195678-96834f50-7de4-11e7-8c73-b8aa7321abfc.gif)